### DISCUSSION TITLE
Save work flow and save template locations?

### DISCUSSION THREAD

**Discussion by bobpuffer1** [Comfy-Org/ComfyUI#1405] [Ideas]
I just want to test my thesis here that workflows shouldn't be saved automatically in the downloads directory but perhaps should be configureable as well as templates shouldn't be stored in the local storage area of the browser. this would seem to be counter to having the comfy UI installation be portable. If there's agreement I'm going to research how to go about accomplishing this and hopefully creating  a PR (after a long while).

**Comment by proceduralworks** [comment]
In my view and to make the app trully production worthy, both templates and workflows locations on local machine should be exposed and their locations should be editable in the settings.

**Comment by sovereignstack** [comment]
IMO, for true portability and peace of mind, workflow that gets saved into the output image should include the data for the templates as well so that it can be recreated on any other machine.
